src/atomic.dex.app.tests.cpp

```diff
@@ -0,0 +1,30 @@
+/******************************************************************************
```

Comment:
This files should not be in this pull request

Reply:
yea, I might have created some files during my testing session and forgot that they will be committed to this pull-request.